brainscore_language/benchmarks/pereira2018_v2022/ceiling_packaging.py

```diff
@@ -0,0 +1,305 @@
+import logging
```

Comment:
this entire file seems to be a copy of `pereira2018/ceiling_packaging.py`, I'm not sure what it's doing